aptos-move/aptos-release-builder/src/utils.rs

```diff
@@ -22,7 +22,46 @@ pub(crate) fn generate_blob(writer: &CodeWriter, data: &[u8]) {
     emit!(writer, "]")
 }
 
-pub(crate) fn generate_governance_proposal_header(writer: &CodeWriter, deps_name: &str) {
+pub(crate) fn generate_next_execution_hash_blob(
+    writer: &CodeWriter,
+    for_address: AccountAddress,
+    next_execution_hash: String,
+) {
+    if next_execution_hash == "vector::empty<u8>()" {
+        emitln!(
+                writer,
+                "let framework_signer = aptos_governance::resolve_multi_step_proposal(proposal_id, @{}, {});\n",
+                for_address,
+                next_execution_hash,
+            );
+    } else {
+        let next_execution_hash_bytes = next_execution_hash.as_bytes();
+        emitln!(
+            writer,
+            "let framework_signer = aptos_governance::resolve_multi_step_proposal("
+        );
+        writer.indent();
+        emitln!(writer, "proposal_id,");
+        emitln!(writer, "@{},", for_address);
+        emit!(writer, "vector[");
+        for (i, b) in next_execution_hash_bytes.iter().enumerate() {
+            if (i + 1) % 20 == 0 {
```

Comment:
Why?

Reply:
not sure if we need this, rn the execution hashes are all the same length and fit in one line but if it ever gets longer then we'll need a new line? but I can delete this